src/components/pipelineRuns-list/PipelineRunsKebab.tsx

```diff
@@ -0,0 +1,210 @@
+import * as React from 'react';
+import { PipelineRunKind, TaskRunKind } from '../../types';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+} from '@patternfly/react-core';
+import { KEBAB_BUTTON_ID } from '../../consts';
+import { useTranslation } from 'react-i18next';
+import {
+  k8sCreate,
+  k8sPatch,
+  useAccessReview,
+  useDeleteModal,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { PipelineRunModel } from '../../models';
+import { returnValidPipelineRunModel } from '../utils/pipeline-utils';
+import { getPipelineRunData } from '../utils/utils';
+import { getTaskRunsOfPipelineRun } from '../hooks/useTaskRuns';
+import {
+  shouldHidePipelineRunCancel,
+  shouldHidePipelineRunStop,
+} from '../utils/pipeline-augment';
+
+type PipelineRunsKebabProps = {
+  obj: PipelineRunKind;
+  taskRuns: TaskRunKind[];
+};
+
+const PipelineRunsKebab: React.FC<PipelineRunsKebabProps> = ({
+  obj,
+  taskRuns,
+}) => {
+  const { t } = useTranslation();
+  const launchDeleteModal = useDeleteModal(obj);
+  const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
`This action will delete resource from k8s but still the resource can be fetched from Tekton Results`

The above message is missing in delete modal if Tektok Result is installed